ddp-workspace/projects/ddp-dsm-ui/src/app/tissue/tissue.component.ts

```diff
@@ -172,6 +172,11 @@ export class TissueComponent {
           this.currentPatchField = null;
           if (parameterName === 'deleted' && tAlias ===  Statics.TISSUE_ALIAS) {
             this.tissue.deleted = true;
+          } else if (parameterName === 'deleted' && tAlias === Statics.SM_ID_ALIAS) {
+            smIdArray[index].deleted = true;
+            if (this.smIdDuplicate[this.currentSMIDField].has(this.createDuplicateIndex(index))) {
+              this.smIdDuplicate[this.currentSMIDField].delete(this.createDuplicateIndex(index));
+            }
```

Comment:
delete it from the frontend data only if it was successfully deleted from the backend